### PR TITLE
[TEST] - SEND feature

### PR DESCRIPTION
**Pull request by beemi**
### TESTS

- SEND test cases
- SEND tests with invalid format address * exceeded limit error message checks

### Pages

- Added Page in tests  for each view

### Test utils

- Generate random ETH and BTC address for tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -89,6 +89,7 @@
     "@vue/eslint-config-standard": "^5.1.2",
     "@vue/test-utils": "^1.1.0",
     "babel-eslint": "^10.1.0",
+    "bitcoin-address-generator": "^1.0.2",
```
Comment (monokh) [inline package.json:92]: Can we avoid adding a new module for this one? From a security perspective, it's best to keep dependencies as low as possible

Comment (monokh) [reply package.json:92]: I think it should be possible to use just `bitcoinjs-lib` which is already included in the wallet

Comment (monokh) [reply package.json:92]: I found an example: https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/addresses.spec.ts#L13

**Review by monokh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (beemi) [reply package.json]: Removed that dependencies and used existing one

**Review by monokh — APPROVED** [review]

**Pull request merged**